vars/cancelPreviousBuilds.groovy

```diff
@@ -17,12 +17,7 @@ def call() {
     def exec = build.getExecutor()
 
     if (build.number != currentBuild.number && exec != null) {
-      exec.interrupt(
-        Result.ABORTED,
-        new CauseOfInterruption.UserInterruption(
-          "Aborted by #${currentBuild.number}"
-        )
-      )
+      exec.doStop()
```

Comment:
Hmm, but in this case, we do not have any cause message?

Reply:
We have it, however on the side of job which *aborts*, not on the side of job which is being aborted. 

I think, that the message was never displayed on the side of the job being aborted, it was not  working, but maybe I'm wrong. I'll merge this, and we can do another PR for the cause messages if you want to.